cmd/reset.go

```diff
@@ -30,6 +31,13 @@ var resetCmd = &cobra.Command{
 		cloudProvider := viper.GetString("kubefirst.cloud-provider")
 
 		checksMap := viper.Get("kubefirst-checks")
+		if checksMap == nil {
+			in := `# Succesfully reset`
+			out, _ := glamour.Render(in, "dark")
+			fmt.Print(out)
+			progress.Progress.Quit()
+			return nil
```

Comment:
Please use the progress success function

```suggestion
		message := `# Succesfully reset`
		progress.Success(message)
```